package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mondaydotcomorg/react-big-calendar",
-  "version": "0.31.0",
+  "version": "0.33.0",
```

Comment:
you can revert this line, the publish script does that:

```
npm run release minor --otp=893632
```

![image](https://github.com/user-attachments/assets/01704bc0-5f1a-4c68-82be-5a89db42b568)

Reply:
is it a problem tho ?